samples/sample_encode/src/sample_encode.cpp

```diff
@@ -200,7 +202,9 @@ mfxStatus ParseInputString(msdk_char* strInput[], mfxU8 nArgNum, sInputParams* p
     pParams->FileInputFourCC = MFX_FOURCC_I420;
     pParams->EncodeFourCC = MFX_FOURCC_NV12;
     pParams->nPRefType = MFX_P_REF_DEFAULT;
+#if (MFX_VERSION >= 1027)
```

Comment:
I had similar intent, but this would make samples dependent from library. We would need at least restructure code in a such way so samples along with mfx_config.h would be detachable from whole tree.